configuration/klippy/beacon_adaptive_heat_soak.py

```
@@ -0,0 +1,751 @@
+								level_2_moving_average_hold_count += 1
+							else:
+								level_2_moving_average_hold_count = 0
+
+							if level_2_moving_average_hold_count >= level_2_moving_average_target_hold_count:
+								level_2_moving_average_trend_checks_passed = all(
+									self._check_trend_projection(
+										level_2_moving_average_history, level_2_moving_average_history_times,
+										trend_check[0], trend_check[1], threshold
+									) for trend_check in level_2_moving_average_trend_checks)
+
+						if moving_average_trend_checks_passed == True or level_2_moving_average_trend_checks_passed == True:
```

Ruff (0.14.7)

Avoid equality comparisons to `True`; use `moving_average_trend_checks_passed:` for truth checks

Replace with `moving_average_trend_checks_passed`

(E712)